ansible_collections/arista/avd/plugins/module_utils/eos_designs_facts.py

```diff
@@ -962,14 +962,27 @@ def bgp_as(self):
         '''
         Get global bgp_as or fabric_topology bgp_as.
 
-        This will fail if none of these are found.
+        This will fail if none of these are found. AS ranges will be expanded and allocated to devices based on:
+         - ID of node (for standalone or A/A MH devices)
+         - ID of node 0 in node group (for MLAG)
+         - Bare ASNs work as per previous behaviour
```

Comment:
Referring to previous behavior in the docstring does not indicate what is the previous behavior :)

probably better to reword this by stating the different cases.